build.sbt

```diff
@@ -1,14 +1,14 @@
 
-scalaVersion := "2.12.4"
+scalaVersion := "2.12.15"
```

Comment:
This is interesting, so we installed 2.13 in previous commits but we were using 2.12?

Reply:
We see the inconsistent versions of scala here. The build file originally contained 2.12.4. And the docker container was being built with 2.13.5.

I just brought them both down to 2.12.latest. I chose 2.12 as it had been around longer and still actively being used (unlike Spark 2.x)